cliquet/tests/support.py

```diff
@@ -87,12 +87,9 @@ class BaseWebTest(FakeAuthentMixin):
 
     api_prefix = "v0"
 
-    def get_test_app(self):
```

Comment:
This is useful to control what's the application used in the tests.

Is it the part that makes things slow? (eg. why do we want to remove this `get_test_app` utility?)